platforms/java/com/area9innovation/flow/HttpSupport.java

```diff
@@ -213,12 +204,30 @@ public final Object httpCustomRequestWithTimeoutNative(String url, String method
 				responseHeaders.add(kv);
 			}
 
+			InputStream inputStream = null;
+			StringBuilder response = new StringBuilder();
+
+			if (responseCode == 200) {
+				inputStream = con.getInputStream();
+			} else {
+				inputStream = con.getErrorStream();
+			}
+
+			if (Objects.nonNull(inputStream)) {
+				String newLine = System.getProperty("line.separator");
+				BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
+				for (String line; (line = reader.readLine()) != null; ) {
+						response.append(line);
+						response.append(newLine);
```

Comment:
For flow it should just always be "\n"... In case there is a configuration issue with java...